src/internal/auth/interceptor.go

```diff
@@ -176,7 +176,7 @@ var authHandlers = map[string]authHandler{
 	"/pps_v2.API/StopPipeline":    authDisabledOr(authenticated),
 	"/pps_v2.API/RunPipeline":     authDisabledOr(authenticated),
 	"/pps_v2.API/RunCron":         authDisabledOr(authenticated),
-	"/pps_v2.API/GetLogs":         authDisabledOr(authenticated),
+	"/pps_v2.API/GetLogs":         authDisabledOr(clusterPermissions(auth.Permission_PIPELINE_GET_LOGS)),
```

Comment:
`GetLogs` is actually a weird RPC because depending on the fields that are set it can be "get the logs for pachd" or "get the logs for the pipelines", which have different access controls. I think it would be reasonable to split those into two different RPCs, and use the interceptor for the cluster logs one, or you could add the auth check in the PPS api server itself.

Reply:
@actgardner to clarify, for now we just want to guard the "get logs for pachd" with perm. is that right?

Reply:
Correct